Civi/RemoteParticipant/Event/GetCreateParticipantFormEvent.php

```diff
@@ -55,6 +55,7 @@ public function __construct($params, $event)
                              'type'        => 'Select',
                              'options'     => [1 => $l10n->ts('Accept Invitation'),
```

Comment:
Usually the values of a multi-value array are either in one line, or on different lines, i.e. there should be a line break after `[` and before `]`.